workflows-scripts/data-export/export-marc-bib-records/README.md

```diff
@@ -44,12 +44,10 @@ To measure slowness, look for Exporting MARC Bib records workflow Transaction av
 
 ## Modules required to be enabled as a pre-requisite to run JMeter script:
 ##### Backend:
-- mod-source-record-storage-3.1.4
-- mod-data-export-1.1.1
-- mod-authtoken-2.4.0
-- mod-permissions-5.9.0
-- okapi-2.38.0
+- mod-source-record-storage
```

Comment:
Why are the version numbers removed?

Reply:
Every release, version numbers keep changing. We already have Test Report with the latest version numbers https://wiki.folio.org/pages/viewpage.action?pageId=52134165